activities/creating-affiliations/index.md

```diff
@@ -11,8 +11,6 @@ Other organizations that we collaborate with can become [affiliates](../../gloss
 
 [See our current affiliates](../../organization/affiliates.md).
 
-If you're interested in being affiliated with us, or have any questions, please email us at <info@publiccode.net>.
```

Comment:
Why is this removed? We still want organizations as affiliates, right?

Reply:
Sentence restored and reworded